src/org/usfirst/frc/team2335/robot/subsystems/Ultrasound.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team2335.robot.subsystems;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.Ultrasonic;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class Ultrasound extends Subsystem
+{
+	Ultrasonic backUltra;
+	Ultrasonic frontUltra;
+
+	public Ultrasound()
+	{
+		backUltra = new Ultrasonic(Robot.BACK_PING, Robot.BACK_ECHO);		
+		frontUltra = new Ultrasonic(Robot.FRONT_PING, Robot.FRONT_ECHO);
+		
+		backUltra.setAutomaticMode(true);
+		frontUltra.setAutomaticMode(true);
+	}
+	
+	public double getRangeBack()
+	{
+		return backUltra.getRangeInches();
+	}
+	
+	public double getRangeFront()
+	{
+		return frontUltra.getRangeInches();
+	}
+	
+	public boolean atRange(double range)
+	{	
+		//TODO: Fine tune values
+		return (range - 10) < getRangeBack() && getRangeBack() < (range + 20) ? true : false;
+	}
+	
+	public boolean atZeroInches()
+	{
+		return getRangeFront() > 15 ? true : false;
```

Comment:
The code implies that the name, "atZeroInches", isn't strictly true. We should add documentation explaining the reason and possibly rename this function.